librz/asm/p/asm_cil.c

```diff
@@ -0,0 +1,27 @@
+// SPDX-FileCopyrightText: 2022 wingdeans
+// SPDX-License-Identifier: LGPL-3.0-only
+
+#include <rz_asm.h>
+#include "cil/cil_dis.c"
```

Comment:
is this really necessary? can you just link the cil_dis object with asm_cil in the meson.build file?